### PR TITLE
todo 007 toggle between task statuses

### PR DESCRIPTION
**Pull request by mEWeil**
Given a user opens the TODO app
Then two toggles are present towards the top of the app
And one toggle is labeled "Pending"
And one toggle is labeled "Completed"
And both toggles are switched to the on position by default
And both sections are viewable on the page

Given a user clicks a toggle into the off position
Then the section for the toggle that was clicked, including the heading, is hidden

Given a user has checked boxes for a hidden section and the "Delete" or "Status" buttons are active
When the "Delete" or "Status" button is clicked
Then the action is not performed on checked items in the hidden section

Given a user clicks a toggle into the on position
Then the section for the toggle that was clicked is shown
And any items that were checked before the item was hidden remain checked

https://user-images.githubusercontent.com/108192731/220798664-f3e564ce-00a9-4eba-a1c6-a9eaaf43a562.mov

### CODE REVIEW

File: Frontend/src/components/TaskList.tsx [hunk 1]
```diff
@@ -30,39 +32,67 @@ export default function TaskList() {
...
+            {tasks.map((task) =>
+              task.isCompleted === false ? (
+                <Task
+                  key={task.id}
+                  task={task}
+                  selectedTasks={selectedTasks}
+                  setSelectedTasks={setSelectedTasks}
+                  taskHandler={taskHandler}
+                />
+              ) : null
```
Comment (jay-meister) [inline Frontend/src/components/TaskList.tsx:69]: You don't need a ternary statement here:
```js
              task.isCompleted === false ? (
                <Task
```

You can use `&&` instead:
```js
              !task.isCompleted && (
                <Task
```

Comment (jay-meister) [reply Frontend/src/components/TaskList.tsx:69]: This should be true for the other ternary statements in this component.

File: Frontend/src/interfaces/IHeaderProps.ts [hunk 2]
```diff
@@ -5,4 +5,6 @@ export interface IHeaderProps {
   setTasks: (allTasks: ITask[]) => void;
   selectedTasks: number[];
   taskHandler: () => void;
+  togglePending: boolean;
+  toggleCompleted: boolean;
```
Comment (jay-meister) [inline Frontend/src/interfaces/IHeaderProps.ts:9]: toggle is a verb, so I would expect `toggleCompleted` to update update some piece of state. 

I would change these names to `showCompleted` & `showPending`.

**Review by jay-meister — COMMENTED** [review]
Looks good!! Just a few small comments

File: Frontend/src/components/TaskList.tsx (hunk 1 above)
Comment (mEWeil) [reply Frontend/src/components/TaskList.tsx:69]: Done.

File: Frontend/src/interfaces/IHeaderProps.ts (hunk 2 above)
Comment (mEWeil) [reply Frontend/src/interfaces/IHeaderProps.ts:9]: Done.

**Pull request merged**